front/app/api/comments/useAddCommentToInitiative.ts

```diff
@@ -1,7 +1,7 @@
 import { useMutation, useQueryClient } from '@tanstack/react-query';
 import commentKeys from 'api/comments/keys';
 import initiativesKeys from 'api/initiatives/keys';
-import { CLErrorsJSON } from 'typings';
+import { CLErrorsWrapper } from 'typings';
```

Comment:
Not deep enough into this to understand this change but: do we want to change this in other files in the api folder then too?